string stl_path
---
float64[] mi_data
bool success